lib/instance.js

```diff
@@ -572,6 +572,36 @@ module.exports = (function() {
       }).then(function() {
         if (!options.fields.length) return this;
 
+        // create relational data for BelongsTo
+        var relationPromises = [];
+        Object.keys(this.dataValues).forEach(function(dataValue) {
+          if (_.has(self.options.includeMap, dataValue)) {
+            var includeModel = self.options.includeMap[dataValue].model;
```

Comment:
you want them hoisted on a per ```then()``` basis, or to the top of the entire method? relationPromises is used in two function scopes here. I'm assuming by 'pre-,' you mean add 'pre' and 'post' to the variable names or something? Otherwise, what DO you mean by that :smile: